app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -335,6 +340,25 @@ class BrowserFragment : Fragment(), BackHandler {
         }
     }
 
+    private fun assignSitePermissionsRules() {
+        val settings = Settings.getInstance(requireContext())
+
+        val recommendedSettings = SitePermissionsRules(
+            camera = ASK_TO_ALLOW,
+            notification = ASK_TO_ALLOW,
+            location = ASK_TO_ALLOW,
+            microphone = ASK_TO_ALLOW
+        )
+        val rules: SitePermissionsRules? = if (settings.shouldRecommendedSettingsBeActivated) {
```

Comment:
would it make more sense to wrap the method call in this check?

Reply:
I'm giving it non-null value in follow up PR https://github.com/mozilla-mobile/fenix/issues/1166. I did this way to satisfy the requirements of this issue.

Reply:
I have local change for that